NikoBracket/CreateBracketViewController.m

```diff
@@ -61,87 +80,296 @@ - (void)fetchData{
-    [cell.homeTeamLogo setImageWithURL:url2];
     
-    if([game[@"HomeTeamMoneyLine"] intValue] < 0){
-        cell.homeFavorite.text = @"FAVORITE";
-        cell.awayFavorite.text = @"";
+    if (self.roundControl.selectedSegmentIndex == 0) {
+        NSDictionary *game = arr[indexPath.row];
+        
+        cell.awayTeamName.text = self.teams[game[@"AwayTeam"]][@"School"];
+        NSURL *url = [NSURL URLWithString:self.teams[game[@"AwayTeam"]][@"TeamLogoUrl"]];
+        [cell.awayTeamLogo setImageWithURL:url];
+        
+        cell.homeTeamName.text = self.teams[game[@"HomeTeam"]][@"School"];
+        NSURL *url2 = [NSURL URLWithString:self.teams[game[@"HomeTeam"]][@"TeamLogoUrl"]];
+        [cell.homeTeamLogo setImageWithURL:url2];
+        
+        if([game[@"HomeTeamMoneyLine"] intValue] < 0){
+            cell.homeFavorite.text = @"FAVORITE";
+            cell.awayFavorite.text = @"";
+        }
+        else{
+            cell.homeFavorite.text = @"";
+            cell.awayFavorite.text = @"FAVORITE";
+        }
     }
     else{
+        cell.awayTeamName.text = self.teams[arr[indexPath.row][0]][@"School"];
+        NSURL *url = [NSURL URLWithString:self.teams[arr[indexPath.row][0]][@"TeamLogoUrl"]];
+        [cell.awayTeamLogo setImageWithURL:url];
+        
+        cell.homeTeamName.text = self.teams[arr[indexPath.row][1]][@"School"];
+        NSURL *url2 = [NSURL URLWithString:self.teams[arr[indexPath.row][1]][@"TeamLogoUrl"]];
+        [cell.homeTeamLogo setImageWithURL:url2];
+        
+        //no odds for theoretical matchups so no favorites
+        cell.awayFavorite.text = @"";
         cell.homeFavorite.text = @"";
-        cell.awayFavorite.text = @"FAVORITE";
     }
+        
 
 }
 
+- (IBAction)roundSelectAction:(id)sender {
+    self.round = self.roundControl.selectedSegmentIndex;
+    
+    [self initializeUserBracket:self.westPicks];
+    [self initializeUserBracket:self.southPicks];
+    [self initializeUserBracket:self.eastPicks];
+    [self initializeUserBracket:self.midwestPicks];
+    [self initializeUserFinalsBracket:self.finalsPicks];
+
+    
+    self.user[@"westPicks"] = self.westPicks;
+    self.user[@"southPicks"] = self.southPicks;
+    self.user[@"eastPicks"] = self.eastPicks;
+    self.user[@"midwestPicks"] = self.midwestPicks;
+    self.user[@"finalsPicks"] = self.finalsPicks;
+    
+    [self.user saveInBackground];
+    [self.createBracketTableView reloadData];
+    
+}
 
 
 - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+   
+    switch (self.roundControl.selectedSegmentIndex) {
+        case 0:
+            return 8;
+            break;
+        case 1:
+            return 4;
+            break;
+        case 2:
+            return 2;
+            break;
+        case 3:
+            return 1;
+            break;
+        case 4:
+            return 2;
+            break;
+        default:
```

Comment:
It would be helpful to have the round numbers as constants at the top of the file (similar to the API endpoint) and replace the usages of them here and throughout this file with the constants. Something like:

```
static int kRound1 = 0;
...
static int kSemiFinal = 4;
static int kFinal = 5;
```